salt/modules/file.py

```diff
@@ -1419,6 +1418,7 @@ def _get_flags(flags):
 
             _flags_acc.append(_flag)
 
+        from functools import reduce  # pylint: disable=redefined-builtin
```

Comment:
It wasn't catching it when it was at the top. It was saying it wasn't defined. But now it's failing lint so I don't know what to do